### DISCUSSION TITLE
dependências em desenvolvimento

### DISCUSSION THREAD

**Discussion by fneto-dev** [HashLoad/boss#117] [Ideas]
É possível adicionar dependências ainda em desenvolvimento?

Em grandes projetos as vezes precisamos mudar uma funcionalidade que está em outro projeto, para as etapas de teste precisamos consumir essa dependência que ainda precisa ser validada para aí sim ter uma TAG dela.

**Comment by viniciussanchez** [comment]
Sim, você pode criar tags de pre-release. Seria meio que versões betas... e ai na hora de instalar pode instalar essas pre-releases do sua dependência. Tirando isso, ainda não temos nada
